agimus_controller/agimus_controller/warm_start_reference.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import numpy.typing as npt
+from typing_extensions import override
+
+import pinocchio as pin
+
+from agimus_controller.warm_start_base import WarmStartBase
+from agimus_controller.trajectory import TrajectoryPoint
+
+
+class WarmStartReference(WarmStartBase):
```

Comment:
Do we add docstring explaining what this warmstart implements?

Reply:
indeed, will add a short one